app/portainer/components/PageHeader/HeaderContent.tsx

```diff
@@ -15,7 +15,7 @@ export function HeaderContent({ children }: PropsWithChildren<unknown>) {
   return (
     <div className="breadcrumb-links">
       <div className="pull-left">{children}</div>
-      {user && (
+      {user && !window.ddExtension && (
```

Comment:
angularjs hides the whole `HeaderContent` while here we hide only the user content